include/language.php

```diff
@@ -151,14 +151,14 @@ class PLL_Language {
 	public $mo_id;
 
 	/**
-	 * ID of the page on front in this language (set from pll_languages_list filter).
+	 * ID of the page on front in this language (set from pll_static_pages filter).
 	 *
 	 * @var int
 	 */
 	public $page_on_front;
```

Comment:
Should we add a default value for this and `page_for_posts`?

Reply:
Done in a58f4f32101ce13d7c4fe75d4cee4aea0f4be7a1